learn/getting_started/quick_start.md

```diff
@@ -269,6 +269,24 @@ If you have multiple indexes, you can switch between them using the indexes drop
 
 ![Meilisearch's search preview indicating the indexes dropdown in the upper right corner](/search_preview/multiple_indexes.png)
 
+## Customization
+
+At this point, you can configure your Meilisearch instance, and customize your index and search responses.  
```

Comment:
Those two `and`s in quick succession bother me a bit. They're not wrong, but perhaps this flows a bit better?

```suggestion
At this point, you can configure your Meilisearch instance and customize your index. When searching, you can use search parameters to refine your results.
```